packages/report-server/src/reportBuilder/functions/basic.ts

```diff
@@ -40,3 +40,7 @@ export const exists = (val: any): boolean => {
 export const notExists = (val: FieldValue): boolean => {
   return val === undefined;
 };
+
+export const length = (val: any[]): number => {
+  return val.length;
+};
```

Comment:
You're written so many tests already, so I hate to press for even more, but could we get some unit tests for this in the function unit tests: `src/__tests__/reportBuilder/functions.test.ts`

I know it gets tests by the other higher level insert tests, but I think the idea is that all these functions in report-server may migrate into the `expression-parser` package soon enough, and if so having tests in place for all of them will make that migration a lot easier 👍

Reply:
I've added a unit test to that file. I don't think it's really a complicated enough function to require plural tests